src/seedu/addressbook/commands/FindNonPrivatePhonesCommand.java

```diff
@@ -0,0 +1,46 @@
+package seedu.addressbook.commands;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+/**
+ * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Keyword matching is case sensitive.
+ */
+public class FindNonPrivatePhonesCommand extends Command {
+
+    public static final String COMMAND_WORD = "findnpphones";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose phone numbers are"
```

Comment:
Can consider adding this into HelpCommand class so that this feature will show up when the user enters the help command